File: AMS_基础信息组件/code/server/source/com.yss.ams.base.information.support/src/main/java/com/yss/ams/base/information/support/bi/orgmgr/pojo/OrgMgr.java
package com.yss.ams.base.information.support.bi.orgmgr.pojo;

import com.yss.framework.api.common.co.AuditableParamPojo;

/**
 * 机构结算会员设置    pojo
 * @author 马向峰  拆分  20170531
 *
 */
public class OrgMgr extends AuditableParamPojo {

	/**
	* 会员号 
	*/
	  private String c_MBR_CODE = "";

	/**
	* 机构代码 
	*/
	  private String c_ORG_CODE = "";

	/**
	* 资金账号 
	*/
	  private String c_ACC_CODE = "";

	/**
	* 开户行名称 
	*/
	  private String c_ORG_NAME = "";

	/**
	* 账户代码 
	*/
	  private String c_CA_CODE = "";

	/**
	* 账户名称 
	*/
	  private String c_CA_NAME = "";

	/**
	 * 开始日期
	 */
	private String d_BEGIN ;

	/**
	 * 结束日期
	 */
	private String d_END ;

	/**
	* 描述 
	*/
	private String c_DESC = "";
	
	/**
	* TASK190987:新增“券商代码”,AddBy-Bohua.Gu-2016/02/26
	*/
	private String c_BROKER_CODE = "";

	private static final long serialVersionUID = 1L ;

	
	public String getD_BEGIN() {
		return d_BEGIN;
	}

	public void setD_BEGIN(String dBEGIN) {
		d_BEGIN = dBEGIN;
	}

	public String getD_END() {
		return d_END;
	}

	public void setD_END(String dEND) {
		d_END = dEND;
	}

	public String getC_MBR_CODE() {
		return c_MBR_CODE;
	}

	public void setC_MBR_CODE(String cMBRCODE) {
		c_MBR_CODE = cMBRCODE;
	}

	public String getC_ORG_CODE() {
		return c_ORG_CODE;
	}

	public void setC_ORG_CODE(String cORGCODE) {
		c_ORG_CODE = cORGCODE;
	}

	public String getC_ACC_CODE() {
		return c_ACC_CODE;
	}

	public void setC_ACC_CODE(String cACCCODE) {
		c_ACC_CODE = cACCCODE;
	}

	public String getC_ORG_NAME() {
		return c_ORG_NAME;
	}

	public void setC_ORG_NAME(String cORGNAME) {
		c_ORG_NAME = cORGNAME;
	}

	public String getC_CA_CODE() {
		return c_CA_CODE;
	}

	public void setC_CA_CODE(String cCACODE) {
		c_CA_CODE = cCACODE;
	}

	public String getC_CA_NAME() {
		return c_CA_NAME;
	}

	public void setC_CA_NAME(String cCANAME) {
		c_CA_NAME = cCANAME;
	}

	
	
	public String getC_DESC() {
		return c_DESC;
	}

	public void setC_DESC(String cDESC) {
		c_DESC = cDESC;
	}

	public String getC_BROKER_CODE() {
		return c_BROKER_CODE;
	}

	public void setC_BROKER_CODE(String c_BROKER_CODE) {
		this.c_BROKER_CODE = c_BROKER_CODE;
	}

	public static long getSerialversionuid() {
		return serialVersionUID;
	}

	public String bizPrimeKeyNames() {
		// TODO Auto-generated method stub
		return "c_MBR_CODE";
	}
}

File: AMS_基础信息组件/code/client/source/YssInformation.Support/Bi/organmgr/Pojo/OrgMgr.cs
﻿using FAST.Core.Util;
using FAST.Common.Service.Pojo;
using FAST.Common.Service.Pojo.Base;
using FAST.Core.Exceptions;


using FAST.Core.Context;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Newtonsoft.Json;

namespace YssInformation.Support.Bi.organmgr.Pojo
{
    /// <summary>
    /// 机构结算会员pojo类
    /// </summary>
    public class OrgMgr : AuditableParamPojo
    {
        /// <summary>
        /// 会员号
        /// </summary>
        private string c_MBR_CODE = "";

        /// <summary>
        /// 机构代码
        /// </summary>
        private string c_ORG_CODE = "";

        /// <summary>
        /// 资金账号
        /// </summary>
        private string c_ACC_CODE = "";

        /// <summary>
        /// 开户行名称
        /// </summary>
        private string c_ORG_NAME = "";

        /// <summary>
        /// 账户代码
        /// </summary>
        private string c_CA_CODE = "";

        /// <summary>
        /// 账户名称
        /// </summary>
        private string c_CA_NAME = "";

        /// <summary>
        /// 开始日期
        /// </summary>
        private string d_BEGIN;

        /// <summary>
        /// 结束日期
        /// </summary>
        private string d_END;

        /// <summary>
        /// 描述
        /// </summary>
        private string c_DESC = "";

        /// <summary>
        /// 券商代码
        /// TASK190987:AddBy-Bohua.Gu-2016/02/26
        /// </summary>
        private string c_BROKER_CODE = "";

        /// <summary>
        /// 属性: 会员号 
        /// </summary>
        [JsonProperty(PropertyName = "c_MBR_CODE")]
        public string C_MBR_CODE
        {
            set { c_MBR_CODE = value; }

            get { return c_MBR_CODE; }
        }

        /// <summary>
        /// 属性: 机构代码 
        /// </summary>
        [JsonProperty(PropertyName = "c_ORG_CODE")]
        public string C_ORG_CODE
        {
            set { c_ORG_CODE = value; }

            get { return c_ORG_CODE; }
        }

        /// <summary>
        /// 属性: 资金账号 
        /// </summary>
        [JsonProperty(PropertyName = "c_ACC_CODE")]
        public string C_ACC_CODE
        {
            set { c_ACC_CODE = value; }

            get { return c_ACC_CODE; }
        }

        /// <summary>
        /// 属性: 开户行名称 
        /// </summary>
        [JsonProperty(PropertyName = "c_ORG_NAME")]
        public string C_ORG_NAME
        {
            set { c_ORG_NAME = value; }

            get { return c_ORG_NAME; }
        }

        /// <summary>
        /// 属性: 账户代码 
        /// </summary>
        [JsonProperty(PropertyName = "c_CA_CODE")]
        public string C_CA_CODE
        {
            set { c_CA_CODE = value; }

            get { return c_CA_CODE; }
        }

        /// <summary>
        /// 属性: 账户名称 
        /// </summary>
        [JsonProperty(PropertyName = "c_CA_NAME")]
        public string C_CA_NAME
        {
            set { c_CA_NAME = value; }

            get { return c_CA_NAME; }
        }

        /// <summary>
        /// 属性: 开始日期 
        /// </summary>
        [JsonProperty(PropertyName = "d_BEGIN")]
        public string D_BEGIN
        {
            set { d_BEGIN = value; }

            get { return d_BEGIN; }
        }

        /// <summary>
        /// 属性: 结束日期 
        /// </summary>
        [JsonProperty(PropertyName = "d_END")]
        public string D_END
        {
            set { d_END = value; }

            get { return d_END; }
        }

        /// <summary>
        /// 属性: 描述 
        /// </summary>
        [JsonProperty(PropertyName = "c_DESC")]
        public string C_DESC
        {
            set { c_DESC = value; }

            get { return c_DESC; }
        }

        /// <summary>
        /// 属性: 券商代码 
        /// TASK190987:AddBy-Bohua.Gu-2016/02/26
        /// </summary>
        [JsonProperty(PropertyName = "c_BROKER_CODE")]
        public string C_BROKER_CODE
        {
            set { c_BROKER_CODE = value; }

            get { return c_BROKER_CODE; }
        }

    }
}




